Rakefile

```diff
@@ -22,3 +24,8 @@ load 'rails/tasks/statistics.rake'
 Bundler::GemHelper.install_tasks
 
 Dir['lib/tasks/*.rake'].each { |r| load r }
+
+spec = Gem::Specification.find_by_name 'bower-rails'
+load "#{spec.gem_dir}/lib/tasks/bower.rake"
```

Comment:
It doesn't work without. Where would the load path come from without requiring it?